app/src/main/cpp/ControllerDelegate.h

```diff
@@ -10,8 +10,10 @@
 #include "vrb/Forward.h"
 #include "Device.h"
 #include "GestureDelegate.h"
+#include "moz_external_vr.h"
```

Comment:
`moz_external_vr.h` should only be included in `ExternalVR.cpp` We don't want gecko types outside of that file.